modules/apps/headless/headless-delivery/headless-delivery-test/src/testIntegration/java/com/liferay/headless/delivery/resource/v1_0/test/KnowledgeBaseArticleResourceTest.java

```diff
@@ -113,6 +164,15 @@ protected KnowledgeBaseArticle randomKnowledgeBaseArticle()
 		KnowledgeBaseArticle knowledgeBaseArticle =
 			super.randomKnowledgeBaseArticle();
 
+		Instant randomInstant = RandomTestUtil.nextDate(
+		).toInstant();
```

Comment:
Please, can you remove the chaining here?

Reply:
Also, if there's only one variable of a kind (e.g. `Instant`) name it simply with the type name (e.g. `Instant instant`).